src/main/java/uk/gov/register/presentation/resource/Pagination.java

```diff
@@ -43,6 +43,23 @@ long offset() {
         return (pageIndex - 1) * pageSize;
     }
 
+    public long getTotalEntries() {
+        return totalEntries;
+    }
+
+    public boolean isFirstPageAndShowsAllEntries() {
```

Comment:
could this have a better name? `isOnlyPage` or `isSolePage` or something like that?